tests/scalers/honeycomb/honeycomb_test.go

```diff
@@ -0,0 +1,308 @@
+//go:build e2e
```

Comment:
How are the metrics sent to honeycomb? I don't see any agent or so that collects them